client/state/a8c-for-agencies/client/actions.ts

```diff
@@ -0,0 +1,60 @@
+import { translate } from 'i18n-calypso';
+// Required for modular state.
+import 'calypso/state/a8c-for-agencies/init';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { NoticeActionOptions } from 'calypso/state/notices/types';
+import {
+	A4A_GET_CLIENT_ERROR,
+	A4A_GET_CLIENT_REQUEST,
+	A4A_GET_CLIENT_SUCCESS,
+} from './action-types';
+import { isFetchingClient } from './selectors';
+import type { APIError, Client, AgencyThunkAction } from '../types';
+
+export function fetchClient( {
+	agencyId,
+	clientId,
+}: {
+	agencyId: string;
+	clientId: string;
+} ): AgencyThunkAction {
+	return ( dispatch, getState ) => {
+		if ( isFetchingClient( getState() ) ) {
+			return;
+		}
+		dispatch( {
+			type: A4A_GET_CLIENT_REQUEST,
+			agencyId,
+			clientId,
+		} );
+	};
+}
+
+export function receiveClient( client: Client ): AgencyThunkAction {
+	return ( dispatch ) => {
+		dispatch( {
+			type: A4A_GET_CLIENT_SUCCESS,
+			client,
+		} );
+	};
+}
+
+export function receiveClientError( error: APIError ) {
+	return (
+		dispatch: ( arg0: {
```

Comment:
Why did we choose `arg0` as the name?

Reply:
It is just a name. I have updated it here: https://github.com/Automattic/wp-calypso/pull/91321/commits/d094a94eee32e385ac6e16b4bc56d7a9b92e0ac9